tests/test_range.py

```diff
@@ -103,6 +103,69 @@ def test_implicit(self):
                           datetime(2017,  6,  16, 11,  0,  0),
                           datetime(2017,  6,  16, 12,  0,  0))
 
+
+    def test_context(self):
+        self.assert_range('2017',
+                          datetime(2017,  1,  1),
+                          datetime.now(),
+                          context=CONTEXT_PAST)
+
+        self.assert_range('2017',
+                          datetime.now(),
+                          datetime(2018,  1,  1),
```

Comment:
I don't know how date time works, so checking if it should stop at 2017.12.31? We shouldn't include Jan 1 2018 in the 2017 year. This appears in the rest of the test cases as well.

Reply:
It's excluded.